From e6c338765d4f68e78d01113ecaf36ca11c8e00f2 Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 9 Sep 2014 14:27:57 +0000
Subject: [PATCH 0496/1000] MANTIS 0025026 Mantis 0025026 extension BoKa Report
 process counters to logging service

platform.idl
V1.3

- extension in PlatformIF interface, to be able to report changes in currentNoOfDocs to logService

git-svn-id: svn://localhost/SelfServiceCommon/trunk@720 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index f57950f8..d51fb340 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "1.9";
+    const string interface_version = "2.0";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -330,6 +330,8 @@ module PlatformManagement
         void dataReceived(in string componentType, in any data);
         
         void setupVCompListener(in string appRef, in string compRef, in any action, in any eventAcquireFilter, in any eventListener, in any elud);
+		
+		void logCounterLevel(in string unitID, in LoggingClass loggingClass, in long currentNoOfDocs, in long mediaStatus);
     };
 };
 
-- 
2.41.0.windows.1

